deploy/helm/grafana-operator/templates/rbac.yaml

```diff
@@ -9,7 +9,11 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: {{ if not $namespaceScoped }}Cluster{{ end }}Role
 metadata:
+  {{- if $namespaceScoped }}
+  namespace: {{ $operatorNamespace }}
+  {{- else }}
   namespace: {{ . }}
+  {{- end }}
```

Comment:
```suggestion
  {{- if $namespaceScoped }}
  namespace: {{ $operatorNamespace }}
  {{- end }}
```

Reply:
Sure, I'll remove that - it did look a bit off but didn't want to break any opinionated design choice that I didin't understand.